contrib/freshmaker-cli

```diff
@@ -225,6 +244,42 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL.
+    """
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, verify=False, auth=krb_auth)
+    r.raise_for_status()
+    data = r.json()
+    return [
+        advisory["id"]
+        for advisory in data
+        if advisory["security_impact"] in ("Important", "Critical")
+    ]
+
+
+def get_freshmaker_events_by_search_keys(api_url, search_keys):
+    print("Getting Freshmaker events")
+    events = []
+    for search_key in search_keys:
+        url = urllib.parse.urljoin(api_url, f'/api/1/events/?search_key={search_key}')
+        r = requests.get(url, timeout=30)
+        r.raise_for_status()
+        ret = r.json()
+        if ret["items"]:
+           events += ret["items"]
+        else:
+           events+=[{"id": None,
+                     "state": 5,
```

Comment:
Doesn't `5` mean `CANCELED`? This doesn't seem right in this instance.

Reply:
I added in EVENT_STATES the fake state "no event". "state":5 corresponds to this value

Reply:
@mulaievaRH I understand, but 5 means Canceled in Freshmaker:
https://github.com/redhat-exd-rebuilds/freshmaker/blob/40e36b671fe123f2e1427ef89b00fd4896f613f2/freshmaker/types.py#L91-L92

You'll need a value that doesn't overlap.

Reply:
Oh. Understood now. Will be changed.